README.md

```diff
@@ -1,3 +1,280 @@
-Placeholder for possible webpack loader for Twitter Bootstrap
+# bootstrap-loader
 
-Considering changing the name from bootstrap-sass-loader.
+Load Bootstrap styles and scripts in your Webpack bundle. This loader uses SASS to process CSS styles. Bootstrap 3 & 4 are supported.
+
+## Installation
+
+Get it via npm:
+
+```bash
+npm install bootstrap-loader
```

Comment:
Keep same github repo, but new loader name. Seems OK. We should push another version that does some sort of install message or maybe a console message, or maybe just an update in the README.md.